src/OpenIddict.Mvc/OpenIddictController.cs

```diff
@@ -41,20 +39,7 @@ public class OpenIddictController<TUser, TApplication, TAuthorization, TToken> :
                 });
             }
 
-            // Note: authentication could be theorically enforced at the filter level via AuthorizeAttribute
```

Comment:
Note: this part is no longer necessary, as it's automatically done by `OpenIddictProvider.HandleAuthorizationRequest`: when receiving a POST request, we store it in the user session and generate a unique identifier. We then redirect the user to the same page but with the `request_id` appended in the query string, which allows us to use the classic `AuthorizeAttribute` logic.